README.md

```diff
@@ -37,7 +37,8 @@ found in the [Solidity documentation](https://docs.soliditylang.org/en/latest/in
 A "Hello World" program in Solidity is of even less use than in other languages, but still:
 
 ```solidity
-pragma solidity >=0.6.0 <0.8.0;
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
 
 contract HelloWorld {
   function helloWorld() external pure returns (string memory) {
```

Comment:
If we're at it, I'd also update the indentation to match what [our style guide says about it](https://docs.soliditylang.org/en/latest/style-guide.html#indentation).

```suggestion
contract HelloWorld {
    function helloWorld() external pure returns (string memory) {
```